api/registrations/serializers.py

```diff
@@ -192,6 +264,13 @@ class RegistrationDetailSerializer(RegistrationSerializer):
     """
     Overrides NodeSerializer to make id required.
     """
+
+    def __init__(self, *args, **kwargs):
+        remove_fields = ['draft_registration', 'registration_choice', 'lift_embargo']
```

Comment:
Removing fields in a subclass feels like improper encapsulation. Also, I'm not sure if there are side effects to accessing `self.fields` this early (this might bite us when we switch to Django models). How about using base class?

```python
class BaseRegistrationSerializer(Serializer):
    ...

class RegistrationSerializer(BaseRegistrationSerializer):
    draft_registration = ....
    registration_choice = ...

class RegistrationDetailSerializer(BaseRegistrationSerializer):
    id = IDField(..)
```